features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -613,6 +616,10 @@
             <EnableDataProviders>{{oauth.grant_type.uma_ticket.retrieve_uma_permission_info_through_introspection}}</EnableDataProviders>
         </Introspection>
 
+        {% if oauth.redirect_to_requested_redirect_uri is defined %}
+        <RedirectToRequestedRedirectUri>{{oauth.redirect_to_requested_redirect_uri}}</RedirectToRequestedRedirectUri>
```

Comment:
same here also

Reply:
will change to oauth.redirect_to_idp_error_page_on_error

Reply:
fixed